P1Gaming/SkyCore
# Implemented slide show player

TO TEST:
1. Open the new StorySlideShow scene.
2. Click play and see a slideshow of some of our placeholder art, and then it jumps to Main scene at the end.
